imap_processing/ultra/l1b/ultra_l1b_extended.py

```diff
@@ -666,106 +668,62 @@ def get_ctof(tof: np.ndarray, path_length: np.ndarray, type: str) -> NDArray:
     path_length : np.ndarray
         Path length (r) (hundredths of a millimeter).
     type : str
-        Type of event, either "ph" or "ssd".
+        Type of event, either "PH" or "SSD".
 
     Returns
     -------
     ctof : np.ndarray
         Corrected TOF (tenths of a ns).
+    magnitude_v : np.ndarray
+        Magnitude of the particle velocity (tenths of a ns / mm).
```

Comment:
1. mag_v is not currently used for anything aside from tests - I take it mag_v will be used in future functions?

2. Also I think it's just reversed in the comment not the code, but shouldn't this be `mm /tenths of a ns`? Not sure where the '10ths of a...' should come in, but the velocity should be a spatial (mm) over a temporal (ns) unit. 
Below, a comment says it's converted to km/s though, so maybe it should be km/s.